libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -151,8 +151,8 @@ singleton AP_Vehicle method get_time_flying_ms uint32_t
 
 include AP_SerialLED/AP_SerialLED.h
 singleton AP_SerialLED alias serialLED
-singleton AP_SerialLED method set_num_LEDs boolean uint8_t 1 16 uint8_t 0 32
-singleton AP_SerialLED method set_RGB void uint8_t 1 16 uint32_t 0U UINT32_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
+singleton AP_SerialLED method set_num_LEDs boolean uint8_t 1 16 uint8_t 0 AP_SERIALLED_MAX_LEDS
+singleton AP_SerialLED method set_RGB void uint8_t 1 16 int8_t -1 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX uint8_t 0 UINT8_MAX
```

Comment:
This is a malformed range check, an `int8_t` can't support a value of `UINT8_MAX`, should be `INT8_MAX`. The generator will secretly fix this for you, but it's worth having right here either way.

Reply:
fixed, thanks